Lightbox.js

```diff
@@ -34,11 +34,13 @@ var Lightbox = React.createClass({
       friction:      PropTypes.number,
     }),
     swipeToDismiss:  PropTypes.bool,
+    transform:       PropTypes.bool,
```

Comment:
I understand that, but the primary focus of this feature is to add zoom in/zoom out support to images. Everything else is secondary, and probably not even all that important to the user of this component.